examples/Quick_Starts/Automated_Meshing/om6VolumeMesh.json

```diff
@@ -0,0 +1,49 @@
+{
+    "refinementFactor": 1.45,
```

Comment:
Validation issue for @maciej-flexcompute to fix. For the time-being, we can remove this line

Reply:
Issue fixed after I switch to v0.2.0b17